travis/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -ex
+if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then     # pull request
+    bundle exec fastlane test
+elif [[ -z "$TRAVIS_TAG" && "$TRAVIS_BRANCH" == "master" ]]; then  # non-tag commits to master branch
+    bundle exec fastlane test
+    bundle exec fastlane assemble
+elif [[ -z "$TRAVIS_TAG" && "$TRAVIS_BRANCH" =~ ^stable-.* ]]; then # non-tag commits to stable branches
```

Comment:
so the convention is to name release branch starting with stable for alpha?

Reply:
yes, that's what we do for ios projects and it seems to work pretty well:  https://github.com/Sage-Bionetworks/elevateMS/branches

Reply:
Why exclude tags though? Usually releases are marked with git tags with a version number

Reply:
It's because CI is handling all of the versioning.  In this setup every merge to a stable branch is a new build and version.   The CI/fastlane handles the tagging and deployments.  The code for that is in fastlane/Fastfile  (lines 45-47 & 63-65)